src/maxminddb.c

```diff
@@ -1740,36 +1771,47 @@ LOCAL int get_entry_data_list(MMDB_s *mmdb,
 
             offset = entry_data_list->entry_data.offset_to_next;
             while (size-- > 0) {
-                MMDB_entry_data_list_s *entry_data_list_to =
-                    data_pool_get(pool);
-                if (NULL == entry_data_list_to) {
+                size_t index2 = 0;
```

Comment:
The naming is a bit hard to keep track of, but it isn't a big deal. Maybe `key_index` and `value_index` or something would help.

Reply:
Good point. I'll change that.